Sources/SwiftGodot/Core/VariantCollection.swift

```diff
@@ -8,8 +8,14 @@
 @_implementationOnly import GDExtension
 
 /// This represents a typed array of one of the built-in types from Godot
-public class VariantCollection<Element: VariantStorable>: Collection {
-    var array: GArray
+public class VariantCollection<Element: VariantStorable>: Collection, ExpressibleByArrayLiteral {
```

Comment:
By conforming to `ExpressibleByArrayLiteral` a variant collection can be initialized with a literal

```swift
let collection: VariantCollection<Node> = []
let collection2: VariantCollection<Node> = [Node()]
```